tools/read_data_files/main/main.go

```diff
@@ -0,0 +1,83 @@
+package main
+
+import (
+	"bytes"
+	"fmt"
+	"io"
+	"os"
+	"strings"
+	"time"
+
+	"github.com/m3db/m3db/encoding"
+	"github.com/m3db/m3db/encoding/m3tsz"
+	"github.com/m3db/m3db/persist/fs"
+	"github.com/m3db/m3db/tools"
+	"github.com/m3db/m3x/ident"
+	xlog "github.com/m3db/m3x/log"
+
+	"github.com/pborman/getopt"
+)
+
+func main() {
```

Comment:
Possibly out of scope, but it would be nice to have the option to generate a valid CSV file with id, timestamp, dp

Doesn't need to be anything fancy, just printing properly formatted lines would be fine

Reply:
Yeah the output right now isn't meant for anything but human consumption. E.g.

```
{id: servers.<redacted>memory.<..>, dp: {Timestamp:2018-03-07 00:09:00 +0000 UTC Value:2.04443207378e+11}}
```

It'd be nice to ensure we do the right thing with quoting/escaping for both json/csv output so we could pipe to jq/*nix tools, but meh. We can add that if/when we need it.